src/examples/draggable/DropzoneDanger.tsx

```diff
@@ -0,0 +1,254 @@
+  KeyboardSensor,
+  KeyboardCoordinateGetter,
+  DragOverlay,
+  DragStartEvent,
+  DragOverEvent,
+  DragEndEvent,
+  useDraggable,
+  useDroppable
+} from '@dnd-kit/core';
+import {
+  Draggable,
+  DraggableList,
+  Dropzone,
+  IDraggableProps
+} from '@zendeskgarden/react-drag-drop';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { MD } from '@zendeskgarden/react-typography';
+
+interface IColumnItem {
+  id: string;
+  value: string;
+}
+
+interface IDraggableItemProps extends IDraggableProps {
+  value: string;
+  isOverlay?: boolean;
+  isGrabbed?: boolean;
+  tabIndex?: number;
+}
+
+interface IDroppableColumnProps {
+  isActive: boolean;
+  isHighlighted: boolean;
+}
+
+const defaultItems: IColumnItem[] = [
+  { id: '1', value: 'Pear' },
+  { id: '2', value: 'Orange' },
+  { id: '3', value: 'Grapes' }
+];
+
+const coordinateGetter: KeyboardCoordinateGetter = (event, args) => {
+  const { context, currentCoordinates } = args;
+  const { active, droppableRects } = context;
+  const isDraggableList = active?.data?.current?.type === 'draggable';
+
+  if (isDraggableList && ['ArrowDown', 'ArrowUp', 'ArrowRight', 'ArrowLeft'].includes(event.key)) {
+    event.preventDefault();
+
+    const rects = [...droppableRects.values()];
+    const target = rects[rects.length - 1];
+    const deltaX = target.left;
+    const deltaY = target.top;
+
+    switch (event.key) {
+      case 'ArrowRight':
+        if (currentCoordinates.x > deltaX) return undefined;
+
+        return { y: deltaY, x: deltaX };
+      case 'ArrowLeft':
+        if (currentCoordinates.x < deltaX) return undefined;
+
+        return { y: deltaY, x: deltaX };
+    }
+  }
+
+  return undefined;
+};
+
+const announcements: Announcements = {
+  onDragStart({ active }) {
+    return `Picked up draggable item ${active.id}.`;
+  },
+  onDragOver({ active, over }) {
+    if (over?.id) {
+      return `Draggable item ${active.id} is over droppable area ${over.id}. Drop to delete.`;
+    }
+
+    return undefined;
+  },
+  onDragEnd({ active }) {
+    return `Draggable item ${active.id} has been deleted.`;
+  },
+  onDragCancel({ active }) {
+    return `Dragging was cancelled. Draggable item ${active.id} was dropped`;
+  }
+};
+
+const StyledHeading = styled(MD)`
+  margin-bottom: ${p => p.theme.space.xs};
+`;
+
+const StyledDropzone = styled(Dropzone)`
+  min-height: ${p => p.theme.space.base * 34}px;
+`;
+
+const DraggableItem = forwardRef<HTMLDivElement, IDraggableItemProps>((props, ref) => {
+  const { isOverlay, value, tabIndex, ...restProps } = props;
+
+  useEffect(() => {
+    if (isOverlay) {
+      const draggableRef = ref as RefObject<HTMLDivElement>;
+
+      if (draggableRef.current) {
+        draggableRef.current.focus();
+      }
+    }
+  });
+
+  return (
+    <Draggable {...restProps} tabIndex={isOverlay ? -1 : tabIndex} ref={ref}>
+      <Draggable.Grip />
+      <Draggable.Content>
+        <div>{value}</div>
+      </Draggable.Content>
```

Comment:
```suggestion
      <Draggable.Content>{value}</Draggable.Content>
```

remove unnecessary `<div>`